### DISCUSSION TITLE
Make Presenter mode more intuitive

### DISCUSSION THREAD

**Discussion by SelinaBuff** [inovex/scrumlr.io#4533] [Proposals]
**User feedback:** 
When I am in presenter mode and open any note then close it, it keeps showing up opened to other users. I would expect that it will close as well.

**Our Considered Use Case:** 
We deliberately wanted to give users the ability to continue discussing the displayed map while the moderator, for example, returns to the board to add an "Action" card. However, what was mentioned in the feedback aligns more with the "Follow Me" function, where the moderator fully controls what is displayed to users in the current state, including the "exit" of a note view.

**Idea**: ??

**Comment by MiroHofo** [comment]
Yes, your considered case is what I was expecting from this function - to show everything what I am doing in scrumlr (something like screen sharing during meetings). Or add an option to (force) close note details for other users in presenter mode.
